checker/tests/nullness/Issue752.java

```diff
@@ -0,0 +1,42 @@
+package a.b.c;
+
+import org.checkerframework.checker.nullness.qual.*;
+
+public class Issue752 {
+
+    Issue752 field = new Issue752();
+    static Issue752 staticField = new Issue752();
+
+    @RequiresNonNull("java.lang")
+    //:: error: (flowexpr.parse.error)
+    void method1() { }
+
+    @RequiresNonNull("java.lang.String")
```

Comment:
Agreed. Since fixing this is an item on my ongoing agenda, for the purposes of this pull request I added ".class" to these expressions so that they are valid. When I fix that work item, I'll have a test case for the expected error.